gitFaked.sh

```diff
@@ -1,9 +1,33 @@
 #!/bin/sh
+
+function mod() {
+    retVal=$1
+    while [ ${retVal} -gt $2 ]; do
+        retVal=$((retVal-$2))
+    done
+}
+
+function rainbowEcho() {
+tput sc
+while [ $# -eq 1 ]; do
+    for i in $(seq 1 15); do
+        tput rc
+        for j in $(seq 0 "${#1}"); do
+            mod $((i+j)) "15"
+            printf "\033[38;5;${retVal}m${1:$j:1}\033[0m"
+        done
+        sleep .15
```

Comment:
We should really up this sleep call to be safe. Suggestion of maybe 30 seconds?

Reply:
the sleep call is intended to be very small, it's for those quality 🚥

Reply:
Maybe add comment for epilepsy warning? Or perhaps a "safe" mode for sensitive eyes?

Reply:
I haven't thought of that, I'll get that change up soon :tm:

Reply:
Still waiting on the world to change (and also for this change).

Reply:
It's a work in progress. For full transparency, we're stuck on figuring out the best way to ensure peak user experience when using `rainbowEcho` without having to memorize too many essential arguments. Unfortunately, I expect this to be on the drawing board for at least another couple months

Reply:
Because I am paying for the premium monthly subscription, this is very sad to hear this. Perhaps a reward in credit can be distributed to the faithful consumers? Or perhaps a gift card to one of your sponsors, Best Buy or Taco Bell? Either way, I wish the best of luck to you and your team to get this feature finished in a timely manner.